### PR TITLE
Call out batch query for DCGETNAME

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: api/sheets/get_name.md [hunk 1]
```diff
@@ -22,14 +22,19 @@ Given a cell or a column range of cells with [DCIDs](/glossary.md) (unique ident
 
 Before trying this method out, make sure to follow the setup directions in [the main section for Sheets docs](/api/sheets/index.html).
 
-### Example 1: Retrieve the name of a country by its DCID.
+### Example 1: Retrieve the name of a country by its DCID
```
Comment (beets) [inline api/sheets/get_name.md:25]: keep the periods here and below? it's been used this way throughout our docs as we rewrote them with annie.

File: api/sheets/get_name.md [hunk 2]
```diff
@@ -22,14 +22,19 @@ Given a cell or a column range of cells with [DCIDs](/glossary.md) (unique ident
...
 
 ![](/assets/images/sheets/sheets_get_name_cote_d_ivoire_cropped.png)
 
-### Example 2: Retrieve the names of a collection of cell lines.
+### Example 2: Retrieve the names of a collection of cell lines
 
 ![](/assets/images/sheets/sheets_get_name_cell_lines_cropped.png)
 
+>  **NOTE:**
```
Comment (beets) [inline api/sheets/get_name.md:33]: maybe call this "Tip" instead?

**Review by beets — APPROVED** [review]
thanks for the doc update

File: api/sheets/get_name.md (hunk 2 above)
Comment (shifucun) [reply api/sheets/get_name.md]: Done

File: api/sheets/get_name.md (hunk 1 above)
Comment (shifucun) [reply api/sheets/get_name.md]: Done

**Comment by shifucun** [comment]
Thanks!

**Pull request merged**